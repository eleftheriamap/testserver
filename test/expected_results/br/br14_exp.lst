
test/exp/test/gen/br/br14_exp.bin:     file format binary


Disassembly of section .data:

0000000000000000 <.data>:
   0:	d28e0cc6 	mov	x6, #0x7066                	// #28774
   4:	f2a00086 	movk	x6, #0x4, lsl #16
   8:	d2800002 	mov	x2, #0x0                   	// #0
   c:	f2b14002 	movk	x2, #0x8a00, lsl #16
  10:	f80004c2 	str	x2, [x6], #0
  14:	d61f00c0 	br	x6
  18:	8a000000 	and	x0, x0, x0
