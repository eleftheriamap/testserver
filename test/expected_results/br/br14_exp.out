Registers:
X00    = 0000000000000000
X01    = 0000000000000000
X02    = 000000008a000000
X03    = 0000000000000000
X04    = 0000000000000000
X05    = 0000000000000000
X06    = 0000000000047066
X07    = 0000000000000000
X08    = 0000000000000000
X09    = 0000000000000000
X10    = 0000000000000000
X11    = 0000000000000000
X12    = 0000000000000000
X13    = 0000000000000000
X14    = 0000000000000000
X15    = 0000000000000000
X16    = 0000000000000000
X17    = 0000000000000000
X18    = 0000000000000000
X19    = 0000000000000000
X20    = 0000000000000000
X21    = 0000000000000000
X22    = 0000000000000000
X23    = 0000000000000000
X24    = 0000000000000000
X25    = 0000000000000000
X26    = 0000000000000000
X27    = 0000000000000000
X28    = 0000000000000000
X29    = 0000000000000000
X30    = 0000000000000000
PC     = 0000000000047066
PSTATE : -Z--
Non-zero memory:
0x00000000 : 0xd28e0cc6
0x00000004 : 0xf2a00086
0x00000008 : 0xd2800002
0x0000000c : 0xf2b14002
0x00000010 : 0xf80004c2
0x00000014 : 0xd61f00c0
0x00000018 : 0x8a000000
0x00047068 : 0x00008a00
